src/pages/AddPersonalBankAccountPage.tsx

```diff
@@ -65,41 +67,43 @@ function AddPersonalBankAccountPage({personalBankAccount, plaidData}: AddPersona
             shouldShowOfflineIndicator={false}
             testID={AddPersonalBankAccountPage.displayName}
         >
-            <HeaderWithBackButton
-                title={translate('bankAccount.addBankAccount')}
-                onBackButtonPress={exitFlow}
-            />
-            {shouldShowSuccess ? (
-                <ConfirmationPage
-                    heading={translate('addPersonalBankAccountPage.successTitle')}
-                    description={translate('addPersonalBankAccountPage.successMessage')}
-                    shouldShowButton
-                    buttonText={translate('common.continue')}
-                    onButtonPress={() => exitFlow(true)}
+            <FullPageNotFoundView shouldShow={!isUserValidated}>
```

Comment:
We should have shown the verification page here instead. This caused https://github.com/Expensify/App/issues/65498